intro-Linux/ch_getting_started.tex

```diff
@@ -158,6 +167,8 @@ \subsection{Using non-defined variables}
 
 The moral here is: \strong{be very careful to not use empty variables unintentionally}.
 
+\strong{Tip for job scripts: use ``set -e -u'' to avoid using empty variables accidentally.}
```

Comment:
That's a good tip, but you'll need to explain it a bit more: what does `-e` do, what does `-u` do, and give an output example (like for `echo $DOESNOTEXIST`).